Gopkg.lock

```diff
@@ -17,6 +17,14 @@
   revision = "d216395917cc49052c7c7094cf57f09657ca08a8"
   version = "v3.0.0"
 
+[[projects]]
+  branch = "master"
+  digest = "1:6b250e53b3b7b9abd7e62f55875c234d2f8b77b846bff200fecafa2dc09af09a"
+  name = "github.com/MakeNowJust/heredoc"
```

Comment:
Licence is acceptable: https://github.com/MakeNowJust/heredoc/blob/master/LICENSE

Reply:
👍 going to remove this library since it wasn't used.